tests/integration/cmor/_fixes/__init__.py

```diff
@@ -0,0 +1,3 @@
+import iris
+import iris.aux_factory
+import iris.cube  # noqa
```

Comment:
Why do you need the #noqa?

Reply:
flake8 complains the import is not used, and throws an error

Reply:
Why do you need the import if it's not used?

Reply:
because it's needed in a bunch of tests, and rather than importing them in each of the test modules it's better to put it in the init - flake is silly and doesn't think outside the box